pom.xml

```diff
@@ -63,6 +64,74 @@
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
Näitä kommentoituja voisi siivoilla pois, nehän saa versiohallinnasta palauteltua kyllä.